src/components/CurrentDirectors/CurrentDirectors.vue

```diff
@@ -0,0 +1,157 @@
+              <span class="directors-title">{{ title }}</span>
+            </v-col>
+          </v-row>
+
+          <!-- List Content -->
+          <v-row
+            class="people-roles-content py-3"
+            v-for="(orgPerson, index) in currentDirectors"
+            :key="index"
+            no-gutters
+          >
+            <!-- Name + Badge -->
+            <v-col class="text-truncate pr-5">
+              <!-- provide tooltip to display full name if name is longer than 25 chars -->
+              <v-tooltip top :disabled="formatFullName(orgPerson.officer).length < 25" color="primary">
+                <template v-slot:activator="{ on }">
+                  <span v-on="on" class="director-name">{{ formatFullName(orgPerson.officer).toLowerCase() }}</span>
+                </template>
+                <span class="director-name">{{ formatFullName(orgPerson.officer).toLowerCase() }}</span>
+              </v-tooltip>
+            </v-col>
+
+            <!-- Mailing Address -->
+            <v-col class="pr-5">
+              <base-address class="director-detail" :address="orgPerson.mailingAddress" />
+            </v-col>
+
+            <!-- Delivery Address -->
+            <v-col class="pr-5">
+              <p v-if="isSame(orgPerson.mailingAddress, orgPerson.deliveryAddress)"
+                class="director-detail">Same as Mailing Address
+              </p>
+              <base-address v-else class="director-detail" :address="orgPerson.deliveryAddress"/>
+            </v-col>
+
+            <!-- Appoinment Date -->
+            <v-col class="pr-5">
+              <span class="director-detail">{{ orgPerson.roles[0].appointmentDate }} to Current</span>
+            </v-col>
+          </v-row>
+        </div>
+      </div>
+    </v-card>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Mixins } from 'vue-property-decorator'
+import { Getter } from 'vuex-class'
+import { OrgPersonIF } from '@/interfaces'
+import { RoleTypes } from '@/enums'
+import { CommonMixin } from '@/mixins'
+import BaseAddress from 'sbc-common-components/src/components/BaseAddress.vue'
+
+@Component({
+  components: {
+    BaseAddress
+  }
+})
+export default class CurrentDirectors extends Mixins(CommonMixin) {
+  // Declarations for template
+  readonly RoleTypes = RoleTypes
+
+  // Global getters
+  @Getter getPeopleAndRoles!: OrgPersonIF[]
+
+  /** Headers for the person table. */
+  readonly tableHeaders = ['Name', 'Mailing Address', 'Delivery Address', 'Effective Dates']
+
+  /**
+   * Called when component is mounted.
+   */
+  private mounted (): void {}
+
+  /**
+   * Returns the current directors from the people and roles.
+   * @returns array of directors
+   */
+  private get currentDirectors () : OrgPersonIF[] {
+    const directors = this.getPeopleAndRoles
+      .filter(people => people.roles.some(role =>
+        role.roleType.toLowerCase() === 'director' && role.cessationDate === null))
```

Comment:
Please use enum for role type.

Reply:
Can cessation date be '' instead of null? If so, consider checking if falsey instead.

Reply:
Also on line 105 below.

Reply:
So I did that initially but the enum gives me the value as 'Director' but on the role I get 'director' so they weren't equating. I could compare to RoleType.DIRECTOR.toLowerCase() if that would be better. Or do you have a better solution?

Reply:
In regard to the cessation date it was null on the one I was testing but I'll change it to just check for a falsy value as I think that's more robust

Reply:
> So I did that initially but the enum gives me the value as 'Director' but on the role I get 'director' so they weren't equating. I could compare to RoleType.DIRECTOR.toLowerCase() if that would be better. Or do you have a better solution?

How did it get set to "director" ?

Do not take an enum and lower-case it. Maybe the enum should be a lower-case value -- depends how the code uses it (ie, where type is set for the current role).

Reply:
> In regard to the cessation date it was null on the one I was testing but I'll change it to just check for a falsy value as I think that's more robust

Yeah, depends how it's initialized or reset. We're not totally consistent on initializing state string to '' instead of null. See https://github.com/bcgov/business-edit-ui/blob/master/src/store/state/state-model.ts